jekyll/_cci2/insights.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: classic-docs
+title: "Using Insights"
+short-title: "Using Insights"
+description: "Viewing the status of repos and test performance"
+categories: [configuring-jobs]
+order: 41
+---
+
+*[Basics]({{ site.baseurl }}/2.0/basics/) > Using Insights*
+
+This document describes creating and using insights in CircleCI in the following sections:
+
+* TOC
+{:toc}
+
+## Overview
+
+CircleCI Insights shows you how your tests are performing with integrated graphs and a dashboard for performance of all of your code repositories.
```

Comment:
Maybe something like this:
Insights dashboard provides you a snapshot the health of you repos that you are following on CircleCI. At glance you can see median build, median queue, last build, success rate and parallelism for your default branch. Note: If you have configured workflows, graphs are going to show you all the jobs that are getting executed for your default branch.

Reply:
thanks, I changed to "Click the Insights menu item in the CircleCI app to view a dashboard showing the health of all repositories you are following. Median build time, median queue time, last build time, success rate, and parallelism appear for your default branch. **Note:** If you have configured Workflows, graphs display all of the jobs that are being executed for your default branch."